changelog/db.changelog-1.4.xml

```diff
@@ -0,0 +1,104 @@
+<changeSet id ="1" author="Leigh Godson">
+    <tagDatabase tag="v1.4.0" />
+</changeSet>
+<changeSet id="2" author="Leigh Godson">
+        <createTable tableName="dax">
+            <column name="paymentReference" type="VARCHAR(30)">
+                <constraints primaryKey="true" nullable="false" primaryKeyName="daxId_pkey"/>
+            </column>
+            <column name="calculationId" type="INTEGER"/>
+            <column name="paymentPeriod" type="VARCHAR(200)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="totalQuarterlyPayment" type="NUMERIC">
+                <constraints nullable="false"/>
+            </column>
+            <column name="transDate" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="datePublished" type="DATE">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="3" author="Leigh Godson">
+        <createTable tableName="totals">
+            <column name="calculationId" type="INTEGER">
+                <constraints primaryKey="true" nullable="false" primaryKeyName="totalsId_pkey"/>
+            </column>
+            <column name="sbi" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="frn" type="NUMERIC">
+                <constraints nullable="false"/>
+            </column>
+            <column name="agreementNumber" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="claimId" type="INTEGER">
+                <constraints nullable="false"/>
+            </column>
+            <column name="schemeType" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="calculationDate" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="invoiceNumber" type="VARCHAR(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="agreementStart" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="agreementEnd" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="totalAdditionalPayments" type="NUMERIC(15,2)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="totalActionPayments" type="NUMERIC(15,2)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="updated" type="DATE">
+                <constraints nullable="false"/>
+            </column>
+            <column name="datePublished" type="DATE">
+                <constraints nullable="true"/>
```

Comment:
`<constraints nullable="true"/>` not necessary

Reply:
removed as suggested